modules/tracker/models/pdo/TrendModel.php

```diff
@@ -122,10 +125,9 @@ public function getScalars($trendDao, $startDate = null, $endDate = null, $userI
      */
     public function getTrendsGroupByDatasets($producerDao, $onlyKey = false)
     {
-        $sql = $this->database->select()->setIntegrityCheck(false)->from(
-            $this->_name,
-            array('config_item_id', 'test_dataset_id', 'truth_dataset_id')
-        )->where('producer_id = ?', $producerDao->getKey())->distinct();
+        $sql = $this->database->select()->setIntegrityCheck(false)->from(array('t' => 'tracker_trend'), array('trendgroup_id')
+        )->join(array('g' => 'tracker_trendgroup'), 't.trendgroup_id = g.trendgroup_id'
+        )->where('g.producer_id = ?', $producerDao->getKey())->distinct();
```

Comment:
Is this distinct still needed?  My guess is no.  I'd like it removed to increase readability if it isn't needed.